### ISSUE TITLE
Subject Area Categorisation

### ISSUE THREAD

**Issue by lena-pl** [working-title/whats-next#8]
I noticed today that a few subjects are duplicated across different categories; eg "Design" appears in both "Technology" and "Arts", and "Media Studies" appears in both "Arts" and "Humanities". This might not be a problem, because it might make them easier to find, but might be worth addressing.

There are also a few missing subjects/interests, eg "Painting" and "Photography" are missing from "Arts", and "Languages" houses only "Classics" (which might be more at home in "Humanities"), but not "German", "Latin", "Japanese", etc.

There is also the question of whether we might be better off not categorising these areas by school subjects in the first place, but rather as broader "interests" ( Issue #4 ), but I am wondering if we should address the categorisation we have at the moment in the meantime? Having a semi-finished version of the categorisation system we have in place at the moment might make user testing a bit easier (or not?). Please discuss!

**Comment by shadrx** [comment]
Yeah - I think it would be good to change them for the meantime and come up with something better later. It would definitely improve user testing and showing it people!

I could create a google spreadsheet (or copy and use the one we had before) that we could all use to sort out the categories for the NCEA data. Then someone can just look at that and create it in the database either with the admin or by directly editing the JSON.

**Comment by lena-pl** [comment]
Yes, that sounds like a good plan! I'd be happy to look at it/contribute!
